utils/gyb_syntax_support/ExprNodes.py

```diff
@@ -312,7 +312,7 @@
     # dot-self-expr -> expr '.' 'self'
```

Comment:
Update the comment as well, like `dot-self-expr -> expr? '.' 'self'`, otherwise LGTM